lnwallet/btcwallet/signer.go

```diff
@@ -368,6 +369,33 @@ func (b *BtcWallet) SignOutputRaw(tx *wire.MsgTx,
 		return nil, err
 	}
 
+	// In case of a taproot output any signature is always a Schnorr
+	// signature, based on the new tapscript sighash algorithm.
+	if txscript.IsPayToTaproot(signDesc.Output.PkScript) {
```

Comment:
Ah ok, we'll do it depending on the output....def easier UX wise as well since the user can just send in all their inputs. 

Doesn't necessarily need to be a part of this PR (maybe will be added re musig2)

Reply:
With the way things are right now, it _always_ assumes the tapscript sighash, so a user wouldn't be able to generate just normal keyspend signatures here. 

So either way, looks like we'll need _some_ additional arg/enum here to reflect all the possibilities.

Reply:
Ah, you're right, we're missing things to actually support signing with MuSig. I'll think a bit about that. Might make sense to do it in a separate PR (that also pulls in the MuSig library, which this PR does not yet do).

Reply:
Yeah totally agree we should do the musig stuff in another PR.